static/app/views/settings/organizationDeveloperSettings/sentryApplicationDetails.tsx

```diff
@@ -182,8 +182,9 @@ export default class SentryApplicationDetails extends AsyncView<Props, State> {
   handleSubmitSuccess = (data: SentryApp) => {
     const {app} = this.state;
     const {orgId} = this.props.params;
+    const tab = this.isInternal ? 'internal' : 'public';
     const baseUrl = `/settings/${orgId}/developer-settings/`;
-    const url = app ? baseUrl : `${baseUrl}${data.slug}/`;
+    const url = app ? `${baseUrl}?tab=${tab}` : `${baseUrl}${data.slug}/`;
```

Comment:
nit: I don't love the idea of 'tab' being in the query params since we're connecting the way the page handles data with the current design of the page. If we change from this 'tab' redesign, `tab` isn't going to make sense as a query parameter.

All this to say I'd prefer all the mentions of `tab` be changed to `type` other than those functions directly responsible for rendering the tabs on the page.

Reply:
This would involve changing the TS types, and maybe function names as well, so I'll leave it to your discretion.

Reply:
Agreed with Leander her